SignalMessaging/contacts/SystemContactsFetcher.swift

```diff
@@ -253,6 +258,7 @@ public class SystemContactsFetcher: NSObject {
         SwiftAssertIsOnMainThread(#function)
         guard authorizationStatus == .authorized else {
             owsFail("should have already requested contact access")
+            self.delegate?.systemContactsFetcher(self, hasAuthorizationStatus: authorizationStatus)
```

Comment:
Existing bug, but shouldn't we be calling the completion handler before we return?

Reply:
Good point. Fixed.